funnel/models/user.py

```diff
@@ -158,6 +158,10 @@ class User(SharedProfileMixin, UuidMixin, BaseMixin, db.Model):
         db.defer('timezone'),
     ]
 
+    __roles__ = {'all': {'read': {'name', 'fullname', 'created_at'}}}
+
+    __datasets__ = {'related': {'name', 'fullname', 'created_at'}}
```

Comment:
Curious why `name` and `fullname`. The usual pairing is `name` and `title` (standard vocabulary) or `username` and `fullname` (User model-specific vocabulary).

Reply:
Also missing `pickername`, while Org has it.

Reply:
roles.all.read should have all the public fields including aliases, and the dataset can restrict to whatever is being used.